pkg/devicemapper/devmapper.go

```diff
@@ -264,11 +264,6 @@ func UdevWait(cookie *uint) error {
 	return nil
 }
 
-// LogInitVerbose is an interface to initialize the verbose logger for the device mapper library.
-func LogInitVerbose(level int) {
```

Comment:
Will do, sorry for the semi-long rant. I cannot describe in words how silly I find it that `libdm` requires you to register a logging callback in order to understand what an error code of `1` actually means.